apps/dashboard/src/components/workflow-list.tsx

```diff
@@ -55,25 +81,59 @@ export const WorkflowList = () => {
         </TableHeader>
         <TableBody>
           {workflowsQuery.isLoading ? (
-            <>loading</>
+            <>
+              {new Array(limit).fill(0).map((_, index) => (
+                <TableRow key={index}>
+                  <TableCell className="flex flex-col gap-1 font-medium">
+                    <Skeleton className="h-5 w-[20ch]" />
+                    <Skeleton className="h-3 w-[15ch] rounded-full" />
+                  </TableCell>
+                  <TableCell>
+                    <Skeleton className="h-5 w-[6ch] rounded-full" />
+                  </TableCell>
+                  <TableCell>
+                    <Skeleton className="h-5 w-[8ch] rounded-full" />
+                  </TableCell>
+                  <TableCell>
+                    <Skeleton className="h-5 w-[7ch] rounded-full" />
+                  </TableCell>
+                  <TableCell className="text-foreground-600 text-sm font-medium">
+                    <Skeleton className="h-5 w-[14ch] rounded-full" />
+                  </TableCell>
+                </TableRow>
+              ))}
+            </>
           ) : (
             <>
               {workflowsQuery.data.workflows.map((workflow) => (
-                <TableRow>
+                <TableRow key={workflow._id}>
                   <TableCell className="font-medium">
-                    <p>{workflow.name}</p>
-                    <p className="text-foreground-400 font-code text-xs">{workflow._id}</p>
+                    <div className="flex items-center gap-1">
+                      {workflow.origin === WorkflowOriginEnum.EXTERNAL && (
```

Comment:
I have checked my framework workflows and we should base this logic on the `type: 'BRIDGE'` instead:
![Screenshot 2024-10-08 at 23 20 32](https://github.com/user-attachments/assets/a370725d-84e0-48f5-bbd4-257e0db05f9b)

Reply:
let's not change this for a moment, there is a be bug that we need to solve first and then we will see if this requires the update or not